client/types/data.d.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { StoreDescriptor } from '@wordpress/data/build-types/types';
+import { AnyConfig } from '@wordpress/data/src/types';
+
+declare module '@wordpress/data' {
+	type ControlDescriptor = {
+		type: string;
+		storeKey: string | StoreDescriptor< AnyConfig >;
+		actionName: string;
+		args: unknown[];
+	};
+
+	interface Controls {
+		dispatch: (
+			storeKey: string,
```

Comment:
I think it's the other way around

For the input, it's typed `string | StoreDescriptor< AnyConfig >` and for the output, it's a `string`

Reply:
Is it resolved? Seems like the typing is the same as before

Reply:
Sorry, it took me a while to understand the input and output here. Resolved in [9a04a36](https://github.com/Automattic/woocommerce-payments/pull/6256/commits/9a04a36a6fe7759b2ab06a6386615c301e70f128)

Reply:
NP, thanks.

What I did to figure out the types is just dig through the packages inside the `node_modules`, for reference
But I got it mixed up the first time on this field